apps/base-docs/base-camp/docs/frontend-setup/wallet-connectors.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Wallet Connectors
+description: Learn about how wallet connector libraries aggregate wallets and make it easier to connect to them from your app.
+hide_table_of_contents: false
+---
+
+One of the most intimidating tasks when building an onchain app is making that initial connection between your users' wallets, and your app. Initial research often surfaces a bewildering number of wallets, each with their own SDKs, and own methods to manage the connection. Luckily, you don't actually need to manage all of this on your own. There are a number of libraries specialized in creating a smooth and beautiful user experience to facilitate this connection.
+
+---
+
+## Objectives
+
+By the end of this guide you should be able to:
+
+- Identify the role of a wallet aggregator in an onchain app
+- Debate the pros and cons of using a template
+- Scaffold a new onchain app with RainbowKit
+
+---
+
+## Connecting to the Blockchain
+
+One of the many challenging tasks of building a frontend that can interface with your smart contracts is managing the user's connection between your onchain app and their [EOA] wallet. Not only is there an ever-growing suite of different wallets, but users can (and probably should!) use several different addresses within the same wallet app.
+
+[Rainbowkit] is one of several options that makes this a little bit easier by serving as an aggregator of wallets, and handling some of the details of connecting them. Alternatives include [ConnectKit], and [Dynamic], which are both excellent choices as well.
+
+Each of these include customizable UI/UX components for inviting the user to connect, displaying connection status, and selecting which wallet they wish to use.
+
+### Using the Quick Start
+
+If you're just trying to get up and running as quickly as possible, you can use RainbowKit's [quick start] script to scaffold an app from their template, with a single command. If you're using Yarn:
+
+```bash
+yarn create @rainbow-me/rainbowkit
```

Comment:
Why are we recommending this? I mean, what's the advantage to have this vs https://github.com/coinbase/build-onchain-apps, which uses RainbowKit already.

Reply:
We're not recommending it, we're just using it.  For learning purposes, the RainbowKit template is good because it's much lighter and simpler than our onchain-apps template.  But I will be linking to that in the beginning as a much more robust starter!

Reply:
What's the goal for us, show case a Wallet Connector or kickstart a project? Beacuse the code RainbowKit produce which is this one https://github.com/rainbow-me/rainbowkit/tree/main/examples/with-create-react-app/src is very very very light.

At that point, would just the doc be enough? or access any of their CodeSandbox to just see how it plays out?

Reply:
Grabbed some time to dig into it!